Source/IssaGame/attributesComponent.h

```diff
@@ -0,0 +1,54 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Components/ActorComponent.h"
+#include "attributesComponent.generated.h"
+
+
+UCLASS( ClassGroup=(Custom), meta=(BlueprintSpawnableComponent) )
+class ISSAGAME_API UattributesComponent : public UActorComponent
+{
+	GENERATED_BODY()
+
+public:	
+	// Sets default values for this component's properties
+	UattributesComponent();
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
```

Comment:
Replace dirty disgusting tab `/t` characters with glorious master race space characters

Reply:
You are absolved of your sins my child